src/js/controller/base.js

```diff
@@ -149,14 +149,12 @@ Base.prototype.updateSchedule = function(schedule, options) {
 
     options = options || {};
 
+    schedule.set('isAllDay', !!options.isAllDay);
```

Comment:
I think you'd better check whether property is undefined. Because if user don't wan't change it, there is no `isAllDay` property. How about like this style?
```js
if (!util.isUndefined(options.isPending)) {
    schedule.set('isPending', options.isPending);
}
```

Reply:
you are right.
I did not check the other part.
thanks